gprofiler/profilers/java.py

```diff
@@ -443,6 +468,26 @@ def _profile_ap_process(self, ap_proc: AsyncProfiledProcess) -> Optional[StackTo
             logger.info(f"Finished profiling process {ap_proc.process.pid}")
             return parse_one_collapsed(output, process_comm(ap_proc.process))
 
+    def _check_hotspot_error(self, ap_proc):
+        if not os.path.isfile(ap_proc.error_file_host):
+            return
+
+        pid = ap_proc.process.pid
+        logger.info(f"Found Hotspot error log at {ap_proc.error_file_host}")
+        contents = open(ap_proc.error_file_host).read()
+        m = VM_INFO_REGEX.search(contents)
+        if m:
+            logger.error(f"Pid {pid} Hotspot VM info: {m[1]}")
```

Comment:
As later on we want to trigger alerts from these message, I suppose it'll be easier if everything is aggregated as a single string in a single call (incl the `Found HotSpot ...`)

Reply:
Done.